src/compas/robots/model/link.py

```diff
@@ -175,6 +352,48 @@ def __init__(self, name, type=None, visual=[], collision=[], inertial=None, **kw
         self.joints = []
         self.parent_joint = None
 
+    @property
+    def data(self):
+        return {
+            'name': self.name,
+            'type': self.type,  # !!!
```

Comment:
what's the `# !!!` for? because type is a reserved keyword and we're using it anyway? ;)

Reply:
oh, no, i just use '!!!' as a flag for things i want to double check (easy to search for this string in the code).  originally i wasn't sure of the type of `type`, but now that i know it's a string i can just delete my flag.  not fussed about it being a reserved word.

Reply:
😉